Automattic/newspack-ads
# Hotfix: responsive sidebar placements

Just like #281, sidebar placements should also use the responsive strategy for AMP.

## How to test

Similar to #281: 

1. On the master branch, create an ad unit with the sizes: 300x250, 760x120
2. Visit Advertising -> Placements and set the "Article below content" to use the created ad unit
3. Visit a post, inspect the below content ad and observe it's a single <amp-ad /> with both sizes
4. Switch to this branch, refresh and observe it's now one element per size
